core/src/main/java/org/geysermc/geyser/text/MinecraftLocale.java

```diff
@@ -44,15 +46,15 @@ public class MinecraftLocale {
 
     public static final Map<String, Map<String, String>> LOCALE_MAPPINGS = new HashMap<>();
 
-    static {
-        // Create the locales folder
-        File localesFolder = GeyserImpl.getInstance().getBootstrap().getConfigFolder().resolve("locales").toFile();
-        //noinspection ResultOfMethodCallIgnored
-        localesFolder.mkdir();
+    private static final Path LOCALE_FOLDER = GeyserImpl.getInstance().getBootstrap().getConfigFolder().resolve("locales");
 
-        // FIXME TEMPORARY
+    static {
         try {
-            Files.delete(localesFolder.toPath().resolve("en_us.hash"));
+            // Create the locales folder
+            Files.createDirectories(LOCALE_FOLDER);
+            Files.createDirectories(LOCALE_FOLDER.resolve("overrides"));
+            // FIXME TEMPORARY
+            Files.delete(LOCALE_FOLDER.resolve("en_us.hash"));
         } catch (IOException ignored) {
```

Comment:
Not sure if this should be ignored , if it is, i'd add a //no-op comment here

Reply:
Unsure what would be the best approach here. The previous result of .mkdir was ignored. The Files.delete call throws a `NoSuchFileException` if the file does not exist. I suppose the createDirectories calls could be separated from the delete into different try/catch blocks?

Reply:
If they weren't logged before, then I don't think we'd need to introduce a check here. Just add `//no-op` as a comment to the catch then to indicate this is supposed to do nothing (atleast that's what I've seen being done for other cases where something is ignored)